src/site/content/en/animations/animations-guide/index.md

```diff
@@ -161,22 +126,139 @@ body > .sidebar {
+and then remove the property once the changing has stopped.
 
 {% Aside %}
 For more information and examples of correct use of `will-change` read
-[Everything You Need To Know About The CSS will-change Property](https://dev.opera.com/articles/css-will-change-property/).
+[Everything You Need To Know About The CSS `will-change` Property](https://dev.opera.com/articles/css-will-change-property/).
 {% endAside %}
 
-The `will-change` property is well supported in browsers.
-If you need a way to force layer creation in older browsers (most likely Internet Explorer at this point)
-then setting `transform: translateZ(0)` will do so.
+If you need a way to force layer creation in one of the rare browsers that doesn't support
+`will-change` (most likely Internet Explorer at this point),
+you can set `transform: translateZ(0)`.
+
+## Debug slow or janky animations {: #debug }
+
+Chrome DevTools and Firefox DevTools have lots of tools to help you figure out why
+your animations are slow or janky.
+
+### Check if an animation triggers layout {: #layout }
+
+[Before](https://glitch.com/~animation-with-top-left).
+
+[After](https://glitch.com/~animation-with-transform).
+
+#### Chrome DevTools {: #layout-chrome }
+
+1. Open the **Performance** panel.
+1. [Record runtime performance](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference#record-runtime)
+   while your animation is happening.
+1. Inspect the **Summary** tab.
+
+If you see a nonzero value for **Rendering** in the **Summary** tab, it may mean that your
+animation is causing the browser to do layout work.
+
+<figure>
+  <img class="w-screenshot w-screenshot--filled" 
+       src="summary-before.jpg" 
+       alt="TODO">
+  <figcaption>
+    The <a href="https://animation-with-top-left.glitch.me/">animation-with-top-left</a>
+    example causes rendering work.
+  </figcaption>
+</figure>
+
+<figure>
+  <img class="w-screenshot w-screenshot--filled" 
+       src="summary-after.jpg" 
+       alt="TODO">
+  <figcaption>
+    The <a href="https://animation-with-transform.glitch.me/">animation-with-transform</a>
+    example does not cause rendering work.
+  </figcaption>
+</figure>
+
+<!-- TODO(kaycebasques): Add video -->
+
+#### Firefox DevTools {: #layout-firefox }
+
+<!-- TODO(kaycebasques): Follow the Chrome DevTools model above -->
+
+### Check if an animation is dropping frames {: #fps }
+
+1. Open the [**Rendering** tab][rendering] of Chrome DevTools.
+1. Enable the **FPS meter** checkbox.
+
+<!-- TODO(kaycebasques): Need to rephrase how to interpret results
+     because the explanation I've provided is not correct -->
+
+At the top of the **FPS meter** UI you see the label **Frames**. Below
+that you see a value along the lines of `49% 405 (1292 m) dropped of 3114`.
+A high-performance animation will has a high percentage, e.g. `99%`. A
+high percentage means that no frames are being dropped.
```

Comment:
Need to find a good way to phrase this. I think the `49%` value is the key. It tells you that only 49% of the frames were rendered (?). In other words 51% of the frames were dropped. My conceptual understanding is shaky here though so I'm not sure if that's correct or how to phrase it